diagnostics/ENSO_MSE/SCATTER/NCL/scatter_04.ncl

```diff
@@ -0,0 +1,323 @@
+    undef2 = 1.1E+20
+    datain1@_FillValue =  undef2
+;;;  read the new model
+    namein22 = prefix + namein2(ibox)
+    datain2 = asciiread(namein22,(/1, ydim/),"float")
+    datain2@_FillValue =  undef2
+;;;   fill dataX and dataY
+    dataX = new ( (/ xdim+1 /), float, datain1@_FillValue)
+    dataY = new ( (/ xdim+1 /), float, datain1@_FillValue)
+    dataX(0:xdim-1) = datain1(0:xdim-1, 0)
+    dataX(xdim) = datain2(0, 0)
+    dataY(0:xdim-1) = datain1(0:xdim-1, 4) + datain1(0:xdim-1, 5)
+    dataY(xdim) = datain2(0, 4) +  datain2(0, 5) 
+
+;;   printMinMax (dataX, False)
+;;   printMinMax (dataY, False)
+
+;----------------------------------------------------------------------
+; Open a PNG file and define the color table
+;----------------------------------------------------------------------
+;;  colors  = (/"white","black","dodgerblue2","firebrick","forestgreen",\
+;              "dodgerblue2","darkorange2","forestgreen","deeppink2",\
+;              "darkorange2","purple"/)
+;;  colors for the markers 
+
+    colors1  = (/ 2,  3,   2,   3,   2,   3,  4,   2,  3,  4,  \ ;;   10
+                 8,  2,   2,   3,   2,   2,  3,   4,  2,  3, \ ;;   20
+                 4,  8,   2,   2,   2,   3,  4,   2,  3,  2, \  ;;; 30
+                 3,  4,   1,  14,   1,   14,  \                 ;; 36
+                14,  6,  14,  14, 14 /)                  ;;; ERA +  NEW_MODEL
+
+   markers1 =   (/16,  16,  1,  1,  2,  4,  5,  6,  6,  6,  \  ;; 10
+                   6,   7,  8,  8,  9, 10, 10, 10, 11, 11,  \ ;;; 20
+                   11, 11, 12, 13, 14, 14, 14, 15, 15,  0,  \ ;; 30
+                    0,  0,  0,  0,  1,  2,  \  ;;  36
+                    4, 12,  7,  5, 16/)               ;;      ERA + NEW_MODEL
+
+
+    markersizes = (/ 0.02,  0.02, 0.02, 0.02, 0.02, 0.03, 0.03, 0.02, 0.03, 0.02, \ ;; 10
+                     0.02,  0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.03, 0.02, 0.02, \ ;; 20
+                     0.02,  0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.02, 0.03, 0.02,  \ ;; 30
+                     0.02,  0.02, 0.02, 0.02,  0.02, 0.02,  \   ;; 36
+                     0.03,  0.03, 0.03, 0.03,  0.03/)        ;;; ERA + NEW_MODEL
+
+
+     markerthick =  (/   3., 3., 3., 3., 3., 10., 10.,  3., 10., 3., \  ;;  10
+                         3., 3., 3., 3., 3.,  3.,  3., 10.,  3., 3., \ ;;; 20
+                         3., 3., 3., 3., 3.,  3.,  3.,  3., 10., 3.,  \ ;; 30
+                         3., 3., 3., 3., 3., 3.,  \  ! 36
+                        10., 10., 6.,  6., 6./)      ;; ERA + NEW_MODEL
+
+;;  wks = gsn_open_wks ("ps","scatter")
+;;   wks_type = "png"
+;;   wks_type@wkWidth = 1000
+;;   wks_type@wkHeight = 1000 
+   wks_type = "eps"
+;;   nameout2 = wkdir + "/SCATTER/PS/" + nameout(ibox)
+   nameout2 = wkdir +  "/model/PS/" + nameout(ibox)
+   wks = gsn_open_wks( wks_type, nameout2)
+
+;;   wks = gsn_open_wks ("X11", "scatter")
+    gsn_define_colormap(wks, "grads_default")
```

Comment:
remove commented-out lines if not needed

Reply:
deleted